src/installer.py

```diff
@@ -6,17 +6,42 @@
 sys.path.append('deps')
 sys.path.append('../deps')
 import pexpect
+import subprocess
+import urllib.request
 
 import password_type
 
 class Installer:
+    click_rootdir = os.getcwd()
+
+    def needs_custom_images(self) -> bool:
+        if os.uname().machine != "aarch64": # images only compiled for arm64
+            return False
+
+        if not os.path.exists("/usr/bin/getprop"):
```

Comment:
Shouldn't we check the whole `$PATH` given the below doesn't hardcode the executable path?

Reply:
I suppose `shutil.which("getprop")` should be fine for this